File: python/7_kyu/disemvowel_trolls.py
# Disemvowel Trolls
# https://www.codewars.com/kata/52fba66badcd10859f00097e/train/python

vowels = "aeiouAEIOU"

def disemvowel(s):
    s = list(s)
    res = []
    for i in s:
        if i not in vowels:
            res.append(i)
    
    return "".join(res)

File: javascript/7_kyu/disemvowel_trolls.js
// Disemvowel Trolls
// https://www.codewars.com/kata/52fba66badcd10859f00097e/train/python

const vowels = "aeiouAEIOU";

function disemvowel(s) {
    s = s.split('');

    var ans = [];

    for (i = 0; i < s.length; i++) {
        if ( !(vowels.includes(s[i])) ) {
            ans.push(s[i]);
        }
    }

    return ans.join("");
}